controller/lib/core/sensors.cpp

```diff
@@ -105,11 +105,11 @@ void sensors_init() {
   Hal.delay(milliseconds(20));
 
   auto set_zero_level = [](Sensor s) {
-    int sum = 0;
+    float sum = 0;
     for (int i = 0; i < SENSOR_SAMPLES_FOR_INIT; i++) {
       sum += Hal.analogRead(pin_for(s));
     }
-    sensorZeroVals[s] = static_cast<float>(sum) / SENSOR_SAMPLES_FOR_INIT;
+    sensorZeroVals[s] = sum / SENSOR_SAMPLES_FOR_INIT;
```

Comment:
I think this was slightly better were summing into an int.  It's conceivable we could overflow the float's mantissa.